src/components/Console.jsx

```diff
@@ -62,5 +68,6 @@ Console.propTypes = {
 };
 
 Console.defaultProps = {
+  currentCompiledProjectKey: undefined,
```

Comment:
`null` and `undefined` are kind of a weird JavaScript idiosyncrasy and I don’t have a great mental model for when to use which, but generally in Popcode I try to avoid ever giving something an explicit value of `undefined`, preferring `null` for cases like this.